### PR TITLE
Updates Community WC meeting to 19:00 UTC

### PR DESCRIPTION
**Pull request by anweshadas**

### CODE REVIEW

File: meetings/README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ You can add this full list of meetings to your personal calendar by importing by
   ([ical](https://raw.githubusercontent.com/ansible/community/main/meetings/ical/network.ics))
   in `#ansible-network`
 
-* [18:00 UTC](http://www.thetimezoneconverter.com/?t=18:00&tz=UTC):
+* [19:00 UTC](http://www.thetimezoneconverter.com/?t=18:00&tz=UTC):
```
Comment (cybette) [inline meetings/README.md:56]: The time in the url needs to be changed too!

Comment (Andersson007) [reply meetings/README.md:56]: @anweshadas ^

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@anweshadas thanks!

**Comment by Andersson007** [comment]
> Thanks @anweshadas for addressing the time change! Can you also make changes to the ics files, similar to what's been done in this PR #672

@cybette isn't it something gotmax is doing in https://github.com/ansible/community/pull/674 ?

**Pull request merged**